server/common/database/migrations/20180828101304-create-course.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const TABLE_NAME = 'course';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable(TABLE_NAME, {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+      allowNull: false
+    },
+    programLevelId: {
+      type: Sequelize.INTEGER,
+      field: 'program_level_id',
+      references: { model: 'program_level', key: 'id' },
+      onDelete: 'NO ACTION',
+      allowNull: false
+    },
+    sourceId: {
+      field: 'source_id',
+      type: Sequelize.INTEGER,
+      allowNull: false
+    },
+    uid: {
+      type: Sequelize.UUID,
+      allowNull: false
+    },
+    schema: {
```

Comment:
It might be nice to set length for items for which you have defined explicit validation rules on model. Also as @vladimyr mentioned bellow, make sure they are in sync with Tailor definitions.